charts/invenio/templates/_helpers.tpl

```diff
@@ -1,10 +1,11 @@
+############################     Redis Hostname     ############################
 {{/*
-Get the redis host name.
+  This template renders the hostname for the Redis instance used.
 */}}
 {{- define "redis.host_name" -}}
```

Comment:
The name of this helper template does not comply with [Helm's recommendations](https://helm.sh/docs/chart_template_guide/named_templates/#declaring-and-using-templates-with-define-and-template):

> Since templates in subcharts are compiled together with top-level templates, it is best to name your templates with chart specific names. A popular naming convention is to prefix each defined template with the name of the chart: `{{ define "mychart.labels" }}`.

It should really be `invenio.redis.hostname` or something similar. I have kept the old name for backwards compatibility, but just tell me and I'll rename it.

Reply:
Will fix it!